src/videotoolbox.cs

```diff
@@ -1167,4 +1232,35 @@ interface VTPixelTransferPropertyKeys {
 		[Field ("kVTPixelTransferPropertyKey_RealTime")]
 		NSString RealTime { get; }
 	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), Watch (9,0), TV (16,0)]
+	[StrongDictionary ("VTPixelRotationPropertyKeys")]
+	interface VTPixelRotationProperties {
+		[Export ("FlipHorizontalOrientation")]
+		bool FlipHorizontalOrientation { get; set; }
+
+		[Export ("FlipVerticalOrientation")]
+		bool FlipVerticalOrientation { get; set; }
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), Watch (9,0), TV (16,0)]
+	[Static]
+	[Advanced]
```

Comment:
What does this attribute do?

Reply:
Here's the attribute definition:

https://github.com/xamarin/xamarin-macios/blob/0d55330059ade28918cef3d5c418a148c557b3c6/src/generator-attributes.cs#L167-L173

It helps to hide this class from the Intellisense until the user enables advanced properties in Visual Studio settings. It's better for users to use the strongly typed StrongDictionary classes than the keys.